drivers/include/sx127x.h

```diff
@@ -216,6 +216,10 @@ typedef struct {
     gpio_t dio5_pin;                   /**< Interrupt line DIO5 (not used) */
 #ifdef SX127X_USE_DIO_MULTI
     gpio_t dio_multi_pin;              /**< Interrupt line for multiple IRQs */
+#endif
+#if defined(SX127X_USE_TX_SWITCH) || defined(SX127X_USE_RX_SWITCH)
```

Comment:
Very minor: I would guard each `rx/tx_switch_pin` attribute of the struct with it's corresponding macro.

Reply:
Ok, I should have thought twice before commenting. I think it's fine like this.